src/assets/svg/external-link.svg

```diff
@@ -1 +1,6 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><g fill="none" fill-rule="evenodd" stroke-linecap="round" stroke="currentColor" stroke-width="2"><path d="M31 19.73v9.732A3.538 3.538 0 0127.462 33H11.538A3.538 3.538 0 018 29.462V13.538A3.538 3.538 0 0111.538 10h9.731"/><path stroke-linejoin="round" d="M26 8h7v7"/></g></svg>
\ No newline at end of file
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Let us remove width="40" height="40". In that way the icon will scale to the parent container and we don't need to override it with CSS.